IanGRap/Squish
# Adding Low Poly Goo

Contributors: Alfredo, Wesley

Purpose: This pull request will add in the Low Poly Goo model + associated textures, along with the correctly mapped materials.

Important files:
Low_Poly_Goo_Rig.fbx
LP_Goo.mat
LP_Skeleton.mat

How to test:
Go to the assets/model folder, click on the Low_Poly_Goo_Rig, and make sure that you can see the model in the bottom right preview window (with eyes and correct coloring).

P.S: Only file has only been added to the project but not the scenes
